migrations/0028-reharvest-automated-record-cards.js

```diff
@@ -0,0 +1,54 @@
+require('dotenv').config();
+
+const axios = require('axios');
+
+const recordOrigin = 'https://api.europeana.eu/record';
```

Comment:
Let's keep this consistent with our use of "origin" elsewhere and make it just "https://api.europeana.eu" with the path included in the URL in `getRecord`.